card/mod/basic_formats/set/all/json.rb

```diff
@@ -139,6 +137,12 @@ def name_search query_attribute: :complete, limit: AUTOCOMPLETE_LIMIT
     }
   end
 
+  view :export_item do
+    item = { name: card.name, type: card.type_name }
+    item[:codename] = card.codename if card.codename
+    item
```

Comment:
not sure this is the best name, but we need a view that gives us only importable attributes (not id, url, etc)